src/Exceptions/HttpInternalServerError.php

```diff
@@ -8,4 +8,3 @@ class HttpInternalServerError extends HttpException {
     public const DESCRIPTION = "Indicates that the server encountered an unexpected condition that prevented it from fulfilling the request.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces